applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.h

```diff
@@ -331,7 +335,12 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ShockCapturingPhysicsBasedProcess :
                 const double Pr_beta_min = 0.9;
                 const double alpha_pr_beta = 2.0;
                 const double mach_threshold = 3.0;
-                const double Pr_beta = Pr_beta_min * (1.0 + std::exp(-2.0 * alpha_pr_beta * (mach - mach_threshold)));
+                double Pr_beta;
+                if (mFarFieldPrandtl == 0.0) {
```

Comment:
```suggestion
                if (mFarFieldPrandtl < std::numeric_limits<double>::epsilon()) {
```
Just in case, this is what @ddiezrod meant.